src/coreclr/System.Private.CoreLib/src/Internal/Runtime/InteropServices/ComActivator.cs

```diff
@@ -91,10 +91,16 @@ public struct ComActivationContext
         public string AssemblyName;
         public string TypeName;
 
+        [RequiresUnreferencedCode("Built-in COM support is not trim compatible")]
```

Comment:
Since we will have https://aka.ms/dotnet-illink/com we should use it here as well and set it to the Url property.